pkg/virt-api/webhooks/validating-webhook/admitters/pod-eviction-admitter.go

```diff
@@ -34,6 +36,18 @@ func NewPodEvictionAdmitter(clusterConfig *virtconfig.ClusterConfig, kubeClient
 	}
 }
 
+func isDryRun(ar *admissionv1.AdmissionReview) bool {
+	dryRun := ar.Request.DryRun != nil && *ar.Request.DryRun == true
+
+	if !dryRun {
+		evictionObject := policyv1.Eviction{}
+		if err := json.Unmarshal(ar.Request.Object.Raw, &evictionObject); err == nil && len(evictionObject.DeleteOptions.DryRun) > 0 {
+			dryRun = evictionObject.DeleteOptions.DryRun[0] == "All"
```

Comment:
The [kubectl docs](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#delete) say that dry-run for delete `Must be "none", "server", or "client".`
How does that map to All or nothing in the go lib?
How did we get a different value when the [docs say](https://github.com/kubevirt/kubevirt/blob/main/vendor/k8s.io/apimachinery/pkg/apis/meta/v1/types.go#L544-L545) only "All" is a valid value?
Also, there's a const for that All: `metav1.DryRunAll`
Thanks

Reply:
`none`, `server` and `client` are allowed value for the kubectl, not the API :)
The dry-run=server is converted in `DryRunAll` by the kubectl (i.e. https://github.com/kubernetes/kubernetes/blob/00c30941260a27e6929aef84c7fdbc8f1508518c/staging/src/k8s.io/kubectl/pkg/cmd/create/create_pdb.go#L216-L218).
+1 for the constant

Reply:
thanks, forgot to remove the constant